ZelBack/config/default.js

```diff
@@ -101,8 +108,10 @@ module.exports = {
       3: 983000, // blockheight. Since this blockheight specification of type 3 is active. User can still submit v1 or v2. UI allows only v2, v3
       4: 1004000, // v4 available, composition
       5: 1142000, // v5 available adding contacts, geolocation
+      6: 1284750, // v6, expiration, app price, t3
     },
     address: 't1LUs6quf7TB2zVZmexqPQdnqmrFMGZGjV6',
+    addressMultisig: 't1UPSwfMYLe18ezbCqnR5QgdJGznzCUYHkj',
```

Comment:
@TheTrunk Isn't the address multisig a T3 address?

Reply:
t3aGJvdtd8NR6GrnqnRuVEzH6MbrXuJFLUX ?

Reply:
Correct as well as blockheight. I have adjusted those temporarily for testing purposes